examples/misc/lib/effective_dart/docs_good.dart

```diff
@@ -31,10 +31,14 @@ void miscDeclAnalyzedButNotTested() {
 
   <Flag>() {
     // #docregion no-annotations
-    /// Defines a flag.
+    /// Defines a flag with the given [name] and [abbr] as abbreviation.
```

Comment:
```suggestion
    /// Creates a new flag.
    ///
    /// Provide a unique [name] for the flag and a short abbreviation ([abbr]).
    /// Both the [name] and [abbr] must be non-empty strings.
    ///
    /// Returns a new flag.
    ///
    /// Throws a [DuplicateFlagException] if a flag with [name] or
    /// [abbr] already exists.
```

Reply:
Or, to be consistent with naming, avoid the abbreviation "abbr". (Slightly ironic to abbreviate "abbreviation" :grin:.)

Reply:
I agree! Maybe change it if you want to.